extensions/datadog/src/datadog-api.ts

```diff
@@ -5,13 +5,14 @@ import { APM } from "./types";
 
 const API_KEY = getPreferenceValues()["api-key"];
 const APP_KEY = getPreferenceValues()["app-key"];
+const API_HOST = getPreferenceValues()["api-host"];
 
 const configuration = v1.createConfiguration({ authMethods: { apiKeyAuth: API_KEY, appKeyAuth: APP_KEY } });
```

Comment:
You also need to update the configuration here.

---
 
**Changing Server**
When talking to a different server, like the eu instance, change the server variables:
```
import { v1 } from '@datadog/datadog-api-client';

const configuration = v1.createConfiguration();

v1.setServerVariables(configuration, {
  site: "datadoghq.eu"
});
```

— [@datadog/datadog-api-client](https://www.npmjs.com/package/@datadog/datadog-api-client)

Reply:
Oh, I didn't see this. I will try to use it. I'll let you know when I finish

Reply:
Done. I would like to get the site from the client configuration to use it in the command-list use cases but I didn't figure out how to do it.